listener.py

```
@@ -1,128 +1,152 @@
-        print("Unable to fetch bot user info.")
-        return
-
-    bot_id = bot.id
-    poll_interval = parse_poll_interval()
-    last_seen_id = None
-    last_checked = datetime.now(timezone.utc) - timedelta(minutes=10)
+    load_env()
+    client = build_client()
+    me = client.get_me().data
+    if not me:
+        raise RuntimeError("Could not resolve authenticated X user for listener")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)